packages/qocamlbrowser/qocamlbrowser.0.2.10/opam

```diff
@@ -3,9 +3,13 @@ maintainer: "kakadu.hafanana@gmail.com"
 homepage: "https://github.com/Kakadu/QOcamlBrowser_quick/"
 bug-reports: "https://github.com/Kakadu/QOcamlBrowser_quick/issues"
 dev-repo: "git://github.com/Kakadu/QOcamlBrowser_quick"
+license: "LGPL-2.1-or-later with OCaml-LGPL-linking-exception"
 authors: "Kakadu"
 synopsis: "OCamlBrowser clone written with OCaml and QtQuick 2"
 
+# Marking as unavailable because on CI file system is readonly and configure fails
+available: false
```

Comment:
Is this just a CI issue? The package would become unavailable for everybody in this way